staging/src/k8s.io/apiserver/pkg/cel/types.go

```diff
@@ -439,37 +441,41 @@ func (rt *DeclTypeProvider) FindDeclType(typeName string) (*DeclType, bool) {
 	return rt.findDeclType(typeName)
 }
 
-// FindFieldType returns a field type given a type name and field name, if found.
+// FindStructFieldNames returns the field names associated with the type, if the type
+// is found.
+func (rt *DeclTypeProvider) FindStructFieldNames(typeName string) ([]string, bool) {
```

Comment:
Is renaming this exported function name essential? It could break out-of-tree projects that depend on this.